### DISCUSSION TITLE
Add a profile for 16xPrompt

### DISCUSSION THREAD

**Discussion by leodip** [netblue30/firejail#6470] [Q&A] [answered]
Hi,

I've not experienced with firejail, and I have tried running this appimage, but failed: 

https://prompt.16x.engineer/download

I'm almost sure it's an electron app.

Can someone help me with a profile / command to run it?

I would like this app to only see one specific folder inside my home directory... and nothing else. Is it possible?

Thanks!

**Comment by rusty-snake** [comment, marked as answer]
Added to #1139
